src/components/nodes/pages/stanford-page/stanford-page-page.tsx

```diff
@@ -34,8 +35,12 @@ const StanfordPagePage = ({node, ...props}: Props) => {
     node.layoutSelection?.id === "stanford_basic_page_full" ||
     !!node.suPageComponents?.find(component => fullWidthComponents.includes(component.__typename))
 
+  const lastItemBanner = node.suPageComponents?.some(
```

Comment:
I think what you want to do is `["sumBanner", "sumCalculator"].includes(node.suPageComponents[node.suPageComponents.length].__typename)`

Reply:
`["sumBanner", "sumCalculator"].includes(node.suPageComponents[node.suPageComponents.length].__typename)` unfortunately throws a couple errors.

I realize while trying to wrangle the snippet to work that we could just check to see if the last array obj is either a Banner or Calculator so I'll update this to reflect that route shortly!